test/gcloud/gcloud_test.rb

```diff
@@ -20,10 +20,11 @@
 describe Gcloud do
   it "calls out to Gcloud.datastore" do
     gcloud = Gcloud.new
-    stubbed_datastore = ->(project, keyfile, scope: nil) {
+    stubbed_datastore = ->(project, keyfile, scope: nil, retries: nil) {
       project.must_equal nil
       keyfile.must_equal nil
```

Comment:
That is failing on only one ruby version and only on Travis-CI. I can't recreate the failure locally. Frustrating.